templates/default/my.cnf.erb

```diff
@@ -19,6 +19,7 @@
 [client]
 port		= <%= node['mariadb']['client']['port']   %>
 socket		= <%= node['mariadb']['client']['socket'] %>
+host		= <%= node['mariadb']['client']['host'] ||= 'localhost' %>
```

Comment:
I think it will be easier to not push conf any if not needed. Perhaps adding an if condition to display it will be better, isn't it ?